web/src/hooks/useTodos.tsx

```diff
@@ -83,7 +83,7 @@ const useTodos = (): [
       .finally(() => setLoading(false))
   }
 
-  return [todos, isLoading, addItem, removeItem, toggleItem, error]
+  return { todos, isLoading, addItem, removeItem, toggleItem, error }
```

Comment:
by returning an object instead of a list, the user of the hook don't have to remember the order of things returned

Reply:
... and they don't have to "get" the `todos`-property in order to get the `isLoading`-property, which is equally as important. They can get the properties they want (via `{ isLoading } = useTodos()`)